### PR TITLE
bazel: Shift flaky test report to bazel

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Commit Message: bazel: Shift flaky test report to bazel
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: ci/flaky_test/BUILD [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+load("@base_pip3//:requirements.bzl", "requirement")
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+py_binary(
+    name = "process_xml",
+    srcs = ["process_xml.py"],
+    deps = [requirement("github")],
```
Comment (phlax) [inline ci/flaky_test/BUILD:12]: requires `repo.PATH`

**Review by wrowe — DISMISSED** [review]
/LGTM deps

**Comment by wrowe** [comment]
Please merge main and we are good to go.

**Review by wrowe — APPROVED** [review]
Examined llnux, windows, osx, looks solid, thanks.

**Pull request merged**